.github/workflows/ci.yml

```diff
@@ -11,6 +11,9 @@ on:
       - main
       - 'release/**'
 
+permissions:
+  contents: write
```

Comment:
Needs `id-token: write` for keyless signing

Reply:
added